### DISCUSSION TITLE
申请图标需要手机有安装邮箱app,是否可以像蓝图一样选择发送到微信上

### DISCUSSION THREAD

**Discussion by qigeco** [zixpo/candybar#203] [Q&A] [answered]
申请图标需要手机有安装邮箱app,是否可以像蓝图一样选择发送到微信上

**Comment by sarsamurmu** [comment, marked as answer]
Check [this configuration](https://github.com/zixpo/candybar-sample/blob/4f701628128d5b563e0839fbc8dcef222eb84118/app/src/main/res/values/dashboard_configurations.xml#L205)

It should be `true` by default
